### ISSUE TITLE
Index list is empty / no create buttons

### ISSUE THREAD

**Issue by styrken** [avo-hq/avo#1708]
### Context
Hi,
I have followed the guides and created a model + resource + policies, but the index pages of the resource is empty and there is no button create new.

What am i doing wrong?

### System configuration
**Avo version**:  2.30.2

**Rails version**: 7.0.4.3

**Ruby version**: 3.0.3p15

**License type**

 - [x] Community
 - [ ] Pro

**Are you using Avo monkey patches or overriding views or view components?**

 - [ ] Yes. If so, please post code samples.
 - [x] No

### Screenshots or screen recordings

<img width="1267" alt="Skærmbillede 2023-04-24 kl  16 18 44" src="https://user-images.githubusercontent.com/2223435/234024587-cc3ae1c0-2182-4ee7-90af-c78c5cd9353e.png">

**Comment by Paul-Bob** [comment]
Hi @styrken. Most probably your policy is not allowing any action. Can you provide the code of your `ApplicationPolicy` and `CompanyPolicy` please? More information about authorization [here](https://docs.avohq.io/2.0/authorization.html)

**Comment by Paul-Bob** [comment]
If you're using the trial make sure you configure it on the `avo.rb` initializer in order to have access to the pro features. Let me know if you have all settled up and still not working.

I suggest you to use inheritance on policies in order to DRY on that classes.

Example:

```ruby
# frozen_string_literal: true

class CompanyPolicy < ApplicationPolicy
  def index?
    true
  end

  def show?
    true
  end

  def create?
    true
  end

  def new?
    true
  end

  def update?
    true
  end

  def edit?
    true
  end

  def destroy?
    true
  end

  class Scope < Scope
    def resolve
      scope.all
    end
  end
end
```

**Comment by Paul-Bob** [comment]
@styrken sorry for the confusion, I just noticed that `view_component` gem released v3.0 today. We still haven't merged a PR that added compatibility for that, we're working on that. Can you use `view_component 2.x` today please? Tomorrow we're releasing `2.31` with a fix for that.

**Comment by styrken** [comment]
@Paul-Bob using `view_components @ 2.82` worked 👍
